executor/chroot/chroot_executor.go

```diff
@@ -66,7 +82,7 @@ func (e *Executor) Run(f def.Formula, j def.Job, d string) def.JobResult {
 }
 
 // Execute a formula in a specified directory. MAY PANIC.
-func (e *Executor) Execute(f def.Formula, j def.Job, d string) def.JobResult {
+func (e *Executor) Execute(f def.Formula, j def.Job, d string, outS, errS io.WriteCloser) def.JobResult {
```

Comment:
Probably depends on if the future involves any more stuff from the `Start` method's context going into the job object construction, so I guess this is season to taste.